### PR TITLE
[doc] Lots of minor updates

### PR DESCRIPTION
**Pull request by pbougue**
No content moves, just text revamp.

* replace `-` by `*` for lists
  (as it is sometimes mistaken for a table of content by vscode)
* remove trailing whitespaces
* replace double-space-newline by `<br>`
* auto-format of tables (by vscode)
* Same as for GTFS, [NTFS] links are deported below

### CODE REVIEW

File: src/documentation/common.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-This document explains the shared parts among all the converters when converting a 
-data set from a given format into a [NTFS](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) dataset.
+This document explains the shared parts among all the converters when converting a
+data set from a given format into a [NTFS] dataset.
+
+[NTFS]: https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md
 
 ## Data prefix
-The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique 
+The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique
```
Comment (woshilapin) [inline src/documentation/common.md:8]: Interesting, I tend to let trailing spaces when it is a paragraph (to mark the fact that it is not the end of the line, only a wrapping for 80 characters)...

Comment (pbougue) [reply src/documentation/common.md:8]: vscode is removing it, and git (git-gui, github) tend to complain about that, so I removed this.
(note that I don't print whitechars in my editor as I've seen you do :wink: )

Comment (datanel) [reply src/documentation/common.md:8]: generally it's not recommended to remove trailing spaces in mardown.

Comment (pbougue) [reply src/documentation/common.md:8]: After discussion, stayed on <br> as it seems to be the only use of trailing-whitespaces so far in md.

**Pull request merged**